diff --git a/node_modules/baileys-pro/lib/Socket/chats.js b/node_modules/baileys-pro/lib/Socket/chats.js
index 8a1b5e9..f1190ca 100644
--- a/node_modules/baileys-pro/lib/Socket/chats.js
+++ b/node_modules/baileys-pro/lib/Socket/chats.js
@@ -689,7 +689,7 @@ const makeChatsSocket = (config) => {
     };
     /** sending non-abt props may fix QR scan fail if server expects */
     const fetchProps = async () => {
-        var _a, _b;
+        var _a, _b, _c;
         const resultNode = await query({
             tag: 'iq',
             attrs: {
